diff --git a/build/config/sysroot.gni b/build/config/sysroot.gni
index 7987e51..1de6942 100644
--- a/build/config/sysroot.gni
+++ b/build/config/sysroot.gni
@@ -14,7 +14,7 @@ declare_args() {

   # Whether to use the default sysroot when building for Linux, if an explicit
   # sysroot isn't set.
-  use_default_linux_sysroot = true
+  use_default_linux_sysroot = false
 }

 if (current_toolchain == default_toolchain && target_sysroot != "") {
